core/stat/base_node.go

```diff
@@ -66,23 +66,18 @@ func (n *BaseStatNode) MinRT() float64 {
 	return float64(n.metric.MinRT())
 }
 
-func (n *BaseStatNode) CurrentGoroutineNum() int32 {
+func (n *BaseStatNode) CurrentConcurrency() int32 {
```

Comment:
How about renaming `goroutineNum` to `concurrency`?

Reply:
make sense